ebpf-io/ebpf.io-website
# feat: changes get-started page

**Describe what changes this pull request brings**
This pull request changes on the get-started page.

**Useful links:**
[Link to page design in Figma](https://www.figma.com/file/DKNPEcutVbDZb1ylyP7wHs/EBPF-Website-Design?type=design&node-id=3409%3A42528&mode=dev)

**Steps to test**
1. Open the [page](https://deploy-preview-467--ebpfio.netlify.app/get-started/)
1. Confirm that everything looks and works as expected

**TODO: before deployment**
- [x] Hide contributor list:
